src/Microsoft.Dnx.Tooling/Restore/RestoreCommand.cs

```diff
@@ -799,7 +825,10 @@ private Task<LockFile> ReadLockFile(string projectLockFilePath)
             }
 
             // Record all libraries used
-            foreach (var item in graphItems.OrderBy(x => x.Match.Library, new LibraryComparer()))
+            var items = graphItems
```

Comment:
Yeah, this was already doing Linq via the order by, plus it wasn't on the runtime path so I just left it as linq.